catroid/src/main/java/org/catrobat/catroid/ui/adapter/SpriteAdapter.java

```diff
@@ -42,213 +46,368 @@
 		View spriteView = convertView;
-		final ViewHolder holder;
+		ViewHolder holder;
+
 		if (convertView == null) {
 			spriteView = inflater.inflate(R.layout.activity_project_spritelist_item, parent, false);
-			holder = new ViewHolder();
-			holder.background = (RelativeLayout) spriteView.findViewById(R.id.spritelist_item_background);
-			holder.checkbox = (CheckBox) spriteView.findViewById(R.id.sprite_checkbox);
-			holder.text = (TextView) spriteView.findViewById(R.id.project_activity_sprite_title);
-			holder.backgroundHeadline = (LinearLayout) spriteView.findViewById(R.id.spritelist_background_headline);
-			holder.objectsHeadline = (LinearLayout) spriteView.findViewById(R.id.spritelist_objects_headline);
-			holder.image = (ImageView) spriteView.findViewById(R.id.sprite_img);
-			holder.scripts = (TextView) spriteView.findViewById(R.id.textView_number_of_scripts);
-			holder.bricks = (TextView) spriteView.findViewById(R.id.textView_number_of_bricks);
-			holder.looks = (TextView) spriteView.findViewById(R.id.textView_number_of_looks);
-			holder.sounds = (TextView) spriteView.findViewById(R.id.textView_number_of_sounds);
-			holder.details = spriteView.findViewById(R.id.project_activity_sprite_details);
+			holder = getHolderViews(spriteView);
+			setOnTouchListener(holder);
 			spriteView.setTag(holder);
 		} else {
-			holder = (ViewHolder) spriteView.getTag();
+			holder = (ViewHolder) convertView.getTag();
 		}
 
-		handleHolderViews(position, holder);
+		setVisibility(spriteView, groupPosition, 0);
 
-		holder.checkbox.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+		if (isGroupSpritePosition(groupPosition)) {
+			setParentHeight(holder, true);
+			setGroupTitle(holder, groupPosition);
+
+			setViewsForExpandedAndCollapsedState(holder, isExpanded);
+			setBackgroundResourcesForGroup(holder);
+			setCheckboxListener(holder, false, false, true);
+			setGroupOnClickListener(holder);
+			handleCheckedSprites(groupPosition, 0, holder);
+			resetNonGroupViews(holder);
+
+			int childrenCount = getChildrenCountOfGroup(groupPosition);
+			holder.scripts.setText(context.getResources().getString(R.string.number_of_objects).concat(" ").concat(Integer.toString(childrenCount)));
+		} else {
+			setParentHeight(holder, false);
+			handleHolderViews(groupPosition, 0, holder);
+			resetGroupViews(holder);
+
+			if (groupPosition == 0 && !spritesListFragment.isBackPackActionMode()) {
+				setBackgroundViewsAndListener(holder);
+			} else {
+				setSpriteViewsAndListener(holder, groupPosition, 0);
+				setOnSpriteEditListenerForGroup(holder, groupPosition);
+			}
+		}
+
+		return spriteView;
+	}
+
+	private void setParentHeight(ViewHolder holder, boolean isGroup) {
+		Resources resources = spritesListFragment.getActivity().getResources();
+		float height = isGroup ? resources.getDimension(R.dimen.spritelist_group_item_height)
```

Comment:
Hmmm, if it is stable at 0.25 I think it wont be a problem, because all Views are moved by 0.25 (can't think of a better way to describe it ^^) Maybe test it with lots of Sprites, like 50 or so, and if you then can't see an issue I think it is fine that way. Just curious: this calculates the height for every item ? So when you have 50 items and for everyone there is an error of 0.1 or something like that you should see that right ?